student.rb

```diff
@@ -0,0 +1,16 @@
+require_relative 'person'
+
+class Student < Person
+  def initialize(classroom)
```

Comment:
- Let's also update our constructor to extend its parent constructor, in our case the class `Person` like this
`def initialize(age, classroom, name = "Unknown", parent_permission = true)`